lib/arrow_web/controllers/api/gtfs_import_controller.ex

```diff
@@ -0,0 +1,30 @@
+defmodule ArrowWeb.API.GtfsImportController do
+  use ArrowWeb, :controller
+  import Ecto.Query
+
+  # TODO: Authorization
+
+  def import(conn, %{"archive" => %Plug.Upload{} = upload}) do
+    version_query =
+      from info in Arrow.Gtfs.FeedInfo, where: info.id == "mbta-ma-us", select: info.version
+
+    version = Arrow.Repo.one!(version_query)
```

Comment:
Note from testing this out:

I changed this to be 
```suggestion
    version = Arrow.Repo.one(version_query)
```

so the initial import can occur (before there's anything in the feed info table).

Reply:
Oop thanks! I had noticed and fixed this yesterday but just got the commit out for it.